bundles/org.openhab.binding.ondilo/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ondilo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Ondilo Bridge Thing Type -->
+	<bridge-type id="bridge">
```

Comment:
Not wrong, but for the sake of clarity, i prefer not to call a bridge `bridge`.

In future there might be differnet types er versions of bridges. And for users the entity type Bridge and it's id bridge might be confusing.

Maybe `gateway`, `account` or if you have anything better.

Reply:
hmm, in majority of the bindings that I'm using the bridge is called bridge:

- automower
- tado
- homematic
- homeconnect calls it `api_bridge`

I could rename it to account if you `insist` on the change.

Reply:
Yes, i think in retrospective those should have been named different too, but it is not worth the breaking change, so for new bindings this would be best to change.

Reply:
renamed `bridge` to `account`